scene/resources/packed_scene.cpp

```diff
@@ -1054,7 +1055,14 @@ Error SceneState::_parse_connections(Node *p_owner, Node *p_node, HashMap<String
 		for (const Node::Connection &F : conns) {
 			const Node::Connection &c = F;
 
-			if (!(c.flags & CONNECT_PERSIST)) { //only persistent connections get saved
+			// Don't save connections that are not persistent.
+			if (!bool(c.flags & CONNECT_PERSIST)) {
+				continue;
+			}
+
+			// Don't include signals that are from scene instances
+			// (they are already saved in the scenes themselves).
+			if (bool(c.flags & CONNECT_INHERITED)) {
```

Comment:
I don't think the `bool` cast is necessary, it's a very common pattern to check bitmasks/flags like this and the results is always used as an inferred boolean check.
```suggestion
			// Don't save connections that are not persistent.
			if (!(c.flags & CONNECT_PERSIST)) {
				continue;
			}

			// Don't include signals that are from scene instances
			// (they are already saved in the scenes themselves).
			if (c.flags & CONNECT_INHERITED) {
```